Libplanet/Action/NullAction.cs

```diff
@@ -0,0 +1,33 @@
+using Bencodex.Types;
+
+namespace Libplanet.Action
+{
+    /// <summary>
+    /// An action implementation which does nothing for filling type parameter taking of
+    /// <see cref="IAction"/>.
+    /// </summary>
+    public class NullAction : IAction
+    {
+        public NullAction()
+        {
+            PlainValue = default(Null);
+        }
+
+        public IValue PlainValue
+        {
+            get;
+            private set;
+        }
+
+        public void LoadPlainValue(
+            IValue plainValue)
```

Comment:
Though it's trivial, but an unnecessary new line.

```suggestion
        public void LoadPlainValue(IValue plainValue)
```